src/wallet/load.cpp

```diff
@@ -49,14 +49,15 @@ bool VerifyWallets(interfaces::Chain& chain, const std::vector<std::string>& wal
         WalletLocation location(wallet_file);
 
         if (!wallet_paths.insert(location.GetPath()).second) {
-            chain.initError(strprintf(_("Error loading wallet %s. Duplicate -wallet filename specified.").translated, wallet_file));
+            chain.initError(strprintf(_("Error loading wallet %s. Duplicate -wallet filename specified."), wallet_file));
             return false;
         }
 
+        // TODO: error_string and warning_string should be bilingual strings.
```

Comment:
I have a slight preference to get #18699 in first, which fixes this TODO

Reply:
Agree.